make/mcu/STM32F3.mk

```diff
@@ -57,7 +57,9 @@ INCLUDE_DIRS    := $(INCLUDE_DIRS) \
 VPATH           := $(VPATH):$(FATFS_DIR)
 endif
 
+ifeq ($(LD_SCRIPT),)
```

Comment:
N.b. there is no `stm32_flash_f303_<size>k_opbl.ld` file available currently, so building an F3 OPBL firmware isn't possible even with this change.